docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: '3.7'
+
+services:
+  web:
+    build: .
+    volumes: &web-volumes
+      - &app-volume .:/app:cached
+      - ~/.ssh:/root/.ssh
+      - ~/.bash_history:/root/.bash_history
+      - &bundle-cache-volume bundle_cache:/bundle_cache
+    ports:
+      - 3000:3000
+      - 3001:3001
+      - 3002:3002
+    depends_on:
+      - db
+    environment: &web-environment
+      DATABASE_HOST: db
+      DATABASE_USERNAME: postgres
+      DATABASE_PASSWORD: postgres
+      BUNDLE_PATH: /bundle_cache
+      GEM_HOME: /bundle_cache
+      GEM_PATH: /bundle_cache
+      RAILS_PORT: 3000
+      RUBYOPT: -W:no-deprecated -W:no-experimental
+    command: bundle exec rails s -b '0.0.0.0' -p 3000
+
+  db:
+    image: postgres:11.4
+    ports:
+      - 5432:5432
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
```

Comment:
it's risky to store this data here (more, you have a public repository!). pls, move it to ENV-file, cuz docker-compose support's it.

[stackoverflow Q&A](https://stackoverflow.com/questions/29377853/how-to-use-environment-variables-in-docker-compose#:~:text=The%20best%20way%20is%20to,with%20the%20new%20environment%20variables)